sequins.go

```diff
@@ -133,13 +133,25 @@ func (s *sequins) initCluster() error {
 		return err
 	}
 
+	ip := ""
+	ipAddresses, err := net.LookupHost(hostname)
+	if err != nil {
+		return err
+	}
+	if len(ipAddresses) != 1 {
+		ip = hostname
+	} else {
+		ip = ipAddresses[0]
+	}
```

Comment:
Oh, and I'm not sure why I missed this earlier (sorry), but I think this can be slightly better phrased as 

```go
ip := hostname
ipAddress, err := net.LookupHost(hostname)
if err != nil {
  return err
}
if len(ipAddresses) == 1 {
  ip = ipAddress[0]
}
```